app/uk/gov/hmrc/trackingconsentfrontend/assets/src/ui/renderBanner.ts

```diff
@@ -1,10 +1,16 @@
 // @ts-ignore
 import bannerHtml from './banner.html';
 
-const renderBanner = () => {
+const renderBanner = (userPreference) => {
     const banner = document.createElement('div')
     banner.className = banner.className + ' cookie-banner';
     banner.innerHTML = bannerHtml
+    banner.querySelectorAll('.acceptAll').forEach(acceptAllButton => {
+        (acceptAllButton).addEventListener('click', e => {
```

Comment:
This won't work on IE8 :-) We should create a utility that determines whether to use attachEvent or addEventListener. Same applies to the DOM ready stuff in index.ts.

Reply:
Fair, this morning's meeting sounds like we don't have to worry about that anymore.  I'd say we come back and fix these things if we start supporting IE8 again.

(for context - at the time you wrote that you're absolutely right and I shouldn't have overlooked it)